openquake/calculators/post_risk.py

```diff
@@ -198,9 +198,31 @@ def execute(self):
             if oq.aggregate_by:
                 set_rlzs_stats(self.datastore, 'agg_curves')
                 set_rlzs_stats(self.datastore, 'agg_losses')
+        return oq.aggregate_by
 
-    def post_execute(self, dummy):
-        pass
+    def post_execute(self, aggregate_by):
+        """
+        Sanity check on tot_losses
+        """
+        logging.info('Sanity check on agg_losses/tot_losses')
+        for kind in 'rlzs', 'stats':
+            agg = 'agg_losses-' + kind
+            tot = 'tot_losses-' + kind
+            if agg not in self.datastore:
+                return
+            if kind == 'rlzs':
+                kinds = ['rlz-%d' % rlz for rlz in range(self.R)]
```

Comment:
Just a note on the variable names: kind and kinds are referring to different entities, may possibly lead to confusion down the line.